supervisor/supervisor.py

```diff
@@ -158,8 +158,16 @@ async def update_apparmor(self) -> None:
                     "Can't update AppArmor profile!", _LOGGER.error
                 ) from err
 
+    @Job(
+        conditions=[
+            JobCondition.FREE_SPACE,
+            JobCondition.HEALTHY,
+            JobCondition.INTERNET_HOST,
+        ],
```

Comment:
We need to be able to ignore this on auto update or add an force option to api. If we rollout ever something that set this flag wrong, we can't update anymore any of our devices. But maybe it's also now fine as we have this technology a long time now and it's seems to work. I don't know

Reply:
I guess Freeze should be on it as well. There are ways to ignore job conditions. I guess that is fine

Reply:
I thought about that. I was basically auditing the conditions and trying to make them consistent across the different update jobs. But I was wondering if this particular one i should leave alone for that exact reason - risk of locking ourselves out of updates. But yea then I remembered job conditions can be ignored.

There is still a small risk. If an issue around these conditions ever makes it out of dev and to beta will be instructing people how to ignore jobsl conditions for a while to get it resolved. Maybe not worth it?

Reply:
that is the golden question. We have to add JobCondition.HEALTHY once if we can pull the min docker API version for the next release at our release files, so people have first to update the OS before it update the supervisor for not break. So the times come to add it, maybe now is fine but we keep the list small as possible, only HEALTHY

Reply:
other thing is now, it raise an SupervisorJobError, we have to make sure everywhere who we use it (not on API) we catch that case if needed

Reply:
I'm going to remove this. It's not really the subject of this PR and it sounds like we do some work around this already so I'll leave changes here for that follow-up work.